pkg/revision/revision.go

```diff
@@ -32,8 +45,17 @@ func init() {
 	if GitCommit == "" {
 		GitCommit = "unknown"
 	}
+	if GoHostOS == "" {
+		GoHostOS = "unknown"
+	}
+	if GoHostArch == "" {
+		GoHostArch = "unknown"
+	}
 	if GoVersion == "" {
-		GoVersion = "go version unknown"
+		// runtime.Version() provides the Go tree's version string at build time
+		// the other values like OS and Arch aren't accessable and are passed in
```

Comment:
```suggestion
		// the other values like OS and Arch aren't accessible and are passed in
```

Reply:
Addressed spelling mistake & updated LDFLAG info at the top of the file